Homework4/DmytroFenevHW4.cpp

```diff
@@ -0,0 +1,215 @@
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <functional>
+#include <iostream>
+
+using Comp = bool (*)(int, int);
+bool Vniz(int a, int b) { return a > b; }
+bool Vverx(int a, int b) { return a < b; }
+
+template <typename T> void sort(T *begin, const T *const end, Comp) {
```

Comment:
я думаю вы немного неверно написали интерфейс функции. 
```suggestion
template <typename T> void sort(T *begin, const T *const end, Comp comp) {
```